### ISSUE TITLE
[SGEN] PCM playback only works when outputting to VGM

### ISSUE THREAD

**Issue by bazz1tv** [bazzinotti/XPMCK#18]
This fact was documented by @mic- in XPMCK's **Implementation specific notes** for Sega Genesis, but I'm interested to know if there's any good reason other than "I just didn't implement PCM playback in the driver yet." Did he try to implement it, and encounter any issues incorporating into the current driver code? Is there some kind of issue there?

EDIT: This is likely related to "The playback library doesn't handle bank switching"

**Comment by mic-** [comment]
> This is likely related to "The playback library doesn't handle bank switching"

Yeah, that's pretty much the reason. I didn't want to deal with the complexity of bank switching in the playback library when I wrote it.
